code/script.js

```diff
@@ -0,0 +1,56 @@
+var flag = false;
+
+// displays numerous inputs (numbers) on the screen
+function displaynumber(n1) {
```

Comment:
Use camel syntax. .... Word will start small letter and another word in property name will start with big letter. ... displayNumber

Reply:
camelCase syntaxing fixed